### PR TITLE
All 6 bces and 5 metrics added

### PR DESCRIPTION
**Pull request by Lord-of-Codes**
Directory structure and code execution style has been changed. 

Readme file needs updation regarding the new developments. yml files removed. 

They'll be added to the readme file.

### CODE REVIEW

File: self_health_check/static_url.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import os
+
+SLACK_URL = os.getenv('SLACK_URL', '')
+BLACKBOX_EXPORTER_URL = 'http://localhost:9115'
```
Comment (kebab-mai-haddi) [inline self_health_check/static_url.py:4]: whats this file?

File: configs/exchanges_config.py [hunk 2]
```diff
@@ -207,47 +154,3 @@
...
-    "Gemini": GEMINI_BTC_TRANSACTION_VOLUME_ABSOLUTE,
-    "Bitfinex": BITFINEX_BTC_TRANSACTION_VOLUME_ABSOLUTE,
-    "Binance": BINANCE_BTC_TRANSACTION_VOLUME_ABSOLUTE,
-    "Coinjar": COINJAR_BTC_TRANSACTION_VOLUME_ABSOLUTE
-}
-
-MEDIAN_SPOT_PRICE_USD = Gauge(
-    'median_spot_price_usd', 'median_spot_price_usd_gauge')
-MEDIAN_SPOT_PRICE_BTC = Gauge(
-    'median_spot_price_btc', 'median_spot_price_btc_gauge')
```
Comment (kebab-mai-haddi) [inline configs/exchanges_config.py]: Why are all these removed?

**Comment by kebab-mai-haddi** [comment]
Thank you @Lord-of-Codes !

**Review by kebab-mai-haddi — COMMENTED** [review]
As of now, this code is running on the server, tried and tested.
Please confirm @Lord-of-Codes by reacting thumbs-up to this comment.

File: configs/exchanges_config.py (hunk 2 above)
Comment (Lord-of-Codes) [reply configs/exchanges_config.py]: These lines are still present in configs/exchanges_config.py. I don't know why these are depicted as 'removed' here.

File: self_health_check/static_url.py (hunk 1 above)
Comment (Lord-of-Codes) [reply self_health_check/static_url.py:4]: Its an old [file](https://github.com/thevantageproject/zcash_service_status_dashboard/blob/640130eea1a6b5de32adbe0be3340f31f28a4a96/self_health_check/config.py). I renamed it such while changing the directory structure. Its function seems redundant and the file can be removed (after making a couple of edits in other files).

**Pull request merged**